bioptim/limits/penalty_option.py

```diff
@@ -167,6 +167,8 @@ def __init__(
         self.transition = False
         self.phase_pre_idx = None
         self.phase_post_idx = None
+        self.states_pre_idx = None
+        self.states_post_idx = None
```

Comment:
I don't think this is relevant in `PenaltyOption`. For me, it only makes sense in `PhaseTransition` which you already added... Therefore, these two lines should be removed, I think